lib/blocksim.py

```diff
@@ -127,6 +127,59 @@ def change_state(self, x):
     def derivative(self, e):
         return 0
 
+class DiscreteTF(blocksim.Block):
+    
+    def __init__(self, name, input_name, output_name, dt, numerator, denominator):
+        """
+        The TF must be of the form: \frac{a_N z^{-N} + ... + a_0 }{b_M z^{-M} + ... + b_0 }.
```

Comment:
It is conventional to call the numerator coefficients b and the denominator a. Perhaps copy from [here](https://docs.scipy.org/doc/scipy/reference/generated/scipy.signal.lfilter.html)